node-runner-cli/api/Api.py

```diff
@@ -1,6 +1,6 @@
 import os
```

Comment:
A really minor / nitpicky point, but each folder seems to have an inconsistent capitalization for its files

Reply:
I have put the filenames with Camelcase where ever they are holding a class with that name. This is followed in api, config, setup modules etc/ 
 As python allows you import functions from different module/file, those files are named snake case like in commands module. I tend to not put any Class definition in these.

Reply:
Ahh okay, gotchya. That makes sense :D